tensorflow/contrib/losses/python/metric_learning/metric_loss_ops.py

```diff
@@ -154,7 +182,7 @@ def masked_minimum(data, mask, dim=1):
   return masked_minimums
 
 
-def triplet_semihard_loss(labels, embeddings, margin=1.0):
+def triplet_semihard_loss(labels, embeddings, margin=1.0, pairwise_dist_fn=pairwise_distance):
```

Comment:
Add pairwise_dist_fn to the list of args

Reply:
Done